src/Controller/Organisation/Delete.php

```diff
@@ -13,7 +13,11 @@
 use Symfony\Component\Routing\RouterInterface;
 use Symfony\Component\Security\Csrf\CsrfToken;
 use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
```

Comment:
Please do not use annotations and the sensio extra bundle.

Reply:
Changes have been made, the extra bundle with IsGranted is gone and replaced by matching routes denied without the right role. But for my knowledge, why is it best not to use this Extra Bundle? Is it to avoid adding annotations?